src/client/components/NotificationList/NotificationList.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { List, ListItem } from 'material-ui/List'
 import Divider from 'material-ui/Divider'
-import { Link } from 'react-router'
+import Link from 'components/Link'
```

Comment:
the changes in this file were done mostly to test using link in another component. I can drop the changes in this file if you wish.